### PR TITLE
Add support for sierra 1.6.0

### LINKED ISSUE #49: Support sierra 1.6.0
2.7.0 will introduce sierra 1.6.0, we need to support it in the USC (at least prepare for the support ATM, when 2.7.0 is in prerelease phase)

### PR DESCRIPTION
**Pull request by tomek0123456789**
Closes #49

### CODE REVIEW

File: src/commands/compile_contract.rs [hunk 1]
```diff
@@ -38,7 +38,7 @@ pub fn compile(mut sierra_json: Value) -> Result<Value> {
 
     let sierra_version = parse_sierra_version(&sierra_json)?;
     match sierra_version.as_slice() {
-        [1, 2..=5, ..] => {
+        [1, 2..=6, ..] => {
```
Comment (piotmag769) [inline src/commands/compile_contract.rs:41]: We should ask compiler team if this is the case

File: src/commands/compile_raw.rs [hunk 2]
```diff
@@ -51,7 +51,7 @@ fn serialize_cairo_program_debug_info(debug_info: &CairoProgramDebugInfo) -> Vec
         .iter()
         .map(|statement_debug_info| {
             (
-                statement_debug_info.code_offset,
+                statement_debug_info.start_offset,
```
Comment (piotmag769) [inline src/commands/compile_raw.rs:54]: Make sure to confirm it by running `snforge` tests (at least some of them) on Scarb `2.7.0-rc.0`

**Review by piotmag769 — COMMENTED** [review]
Would be great to have it by Wednesday (I want to release new profiler together with new snforge)

File: src/commands/compile_raw.rs (hunk 2 above)
Comment (tomek0123456789) [reply src/commands/compile_raw.rs:54]: When run using `statement_info.end_offset`, `snforge` tests fail

**Review by piotmag769 — APPROVED** [review]

**Pull request merged**